packages/react-components/lib/tables/table-footer-pagination.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Mmm.. sorry but maybe I'm missing something, I'm using the `TablePagination` from material UI https://next.material-ui.com/components/pagination/#table-pagination . I just wrapped it into a custom component for two things: 
 * In case we have another table that needs pagination.
 * In case we need to change the paginations of all tables later (because as you said it's in `lab` version).

If none of those reasons convince you, I can put it without problem directly in `log-table.tsx` =)